### ISSUE TITLE
2 warnings during make command

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/contacts#186]
```
/home/jan/nextcloud/apps/contacts/js/services/contact_service.js
   40:10  warning  Unexpected console statement        no-console
  141:46  warning  Don't make functions within a loop  no-loop-func

✖ 2 problems (0 errors, 2 warnings)
```

cc @nextcloud/contacts

**Comment by skjnldsv** [comment]
Is this really an issue?

**Comment by irgendwie** [comment]
Not an issue but a bit ugly, especially if someone inexperienced tries to work on the contacts app.
@Henni @skjnldsv This is also a good time to think about how we want to use the console for debug and info messages in the future.

**Comment by Henni** [comment]
This should be solved, but with a pretty low priority.
A perfect starter issue in my opinion.

We might want to remove the `no-console` rule from eslint for debugging reasons in the future.
Alternatively we should define eslint exceptions in our code. (http://eslint.org/docs/user-guide/configuring.html#configuring-rules)

**Comment by tcitworld** [comment]
Debug shouldn't be in master branch :-)

**Comment by irgendwie** [comment]
The last make problems/warnings were fixed or suppressed by #367
